package.json

```diff
@@ -22,6 +22,7 @@
     "@chakra-ui/system": "^2.3.5",
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
+    "@fontsource/inter": "^4.5.14",
```

Comment:
Do you think using @next/font is better here?
https://nextjs.org/docs/basic-features/font-optimization

Reply:
oh no we're using Next 12 😭 
<img width="678" alt="image" src="https://user-images.githubusercontent.com/19740800/210786895-fb2c6aaa-9765-40cf-8fac-ae7f4068a087.png">

Reply:
i'll try to upgrade now